dist/doc/usage.md

```diff
@@ -18,7 +18,7 @@ serve as the foundation for whatever you're interested in building.
 
 Even the basic use-case of a simple static site can be enhanced by manipulating
 the code through an automated build process. Moving up in complexity HTML5
-Boilerplate can be integrated with whatever front-end framework, CMS or
+Boilerplate can be be integrated with whatever front-end framework, CMS or
```

Comment:
Duplicate? `be `

Reply:
Thanks for picking that up, will fix now - I noticed this was addressed manually in https://github.com/h5bp/html5-boilerplate/commit/37871286b4cb73d828eddf87c3f7f0d2e8b30a68 but that was in `dist/doc/usage.md` directly as opposed to during the build process - will track down the culprit right now.

Reply:
Fixed the error in `src/doc/usage.md` and rebuilt, thus fixing in `dist/doc/usage.md` as well.